### ISSUE TITLE
No method matching species error 

### ISSUE THREAD

**Issue by vkeerth** [ReactionMechanismGenerator/ReactionMechanismSimulator.jl#161]
While trying to run the simulation for Steam methane reforming (examples/rmg/catalysis/methane_steam) example of RMG software using RMS in Jupyter notebook (Julia 1.6.0), the following error always pops up for the code;

Code: 
phaseDict = readinput("rms/chem76.rms")

Error:
MethodError: no method matching Species(; index=23, name="site", adjlist="1 X u0 p0 c0\n\n", radicalelectrons=0, thermo=NASA{EmptyThermoUncertainty}
  polys: Array{NASApolynomial}((2,))
  unc: EmptyThermoUncertainty EmptyThermoUncertainty()
, smiles="[Pt]")

I have attached the word document of the rms file. Can someone guide me?
Chem76 file: 
[chem76.docx](https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl/files/6997967/chem76.docx)

**Comment by vkeerth** [comment]
Yes, thanks for getting back. Please find the pdf file attached. 
[Untitled1_using_Julia.pdf](https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl/files/7356621/Untitled1_using_Julia.pdf)

**Comment by vkeerth** [comment]
Yes, thanks for getting back. Please find the pdf file attached.
[Untitled1_using_Julia.pdf](https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl/files/7356632/Untitled1_using_Julia.pdf)

**Comment by vkeerth** [comment]
RMG version: 3.1.0
pyrms downloaded from: https://github.com/ReactionMechanismGenerator/pyrms

I have attached all the rms files of methane_steam into the zip file below.
[RMS.zip](https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl/files/7356963/RMS.zip)

**Comment by mjohnson541** [comment]
Hmm...I'm not getting this issue on RMS master and I don't think you should be getting this on the latest release v0.4.0 so I suspect your RMS version pyrms has installed is too old. Try running in the command line: 

`julia -e 'using Pkg; Pkg.update("ReactionMechanismSimulator")'`

**Comment by mjohnson541** [comment]
Try opening the julia repl with `julia` and then running
`]add StochasticDiffEq@6.36.0`
